Rebuild-Black-Business/RBB-Website
# Fix/modal overflow

# Describe your PR
The modal for /ally report/unlist had an overflowing header. The issue is with Chakra (I created an issue there)

The proposed fixes were:

- Add vertical spacing
- Add right margin
- add max-width (solution implemented)

![image](https://user-images.githubusercontent.com/8475305/84315424-a51e5180-ab37-11ea-8ecc-1379d0edcdff.png)

Related to #
Fixes #

## Pages/Interfaces that will change
/ally modal

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

![image](https://user-images.githubusercontent.com/8475305/84315569-ddbe2b00-ab37-11ea-8237-7a8988340199.png)

![image](https://user-images.githubusercontent.com/8475305/84315609-e9a9ed00-ab37-11ea-8509-fc5a86ca9740.png)

## Steps to test

1. Go to /ally
2. Accept T&C
3. Click on report/unlink
4. Check if the text goes behind the close button or overflows outside of the modal

### Additional notes

Not sure on other modals!
